zebra-consensus/src/checkpoint.rs

```diff
@@ -781,17 +785,17 @@ mod tests {
             BlockHeight(0)
         );
 
-        /// Make sure the verifier service is ready
+        /// SPANDOC: Make sure the verifier service is ready
```

Comment:
`spandoc` is now configured to ignore all doc comments that don't start with `SPANDOC: `. This is to prevent issues where people accidentally use `///` and `//` interchangably and to help new contributors figure out wtf is going on with our in-function doc comments